config/kraft/broker.properties

```diff
@@ -24,7 +24,7 @@
 process.roles=broker
 
 # The node id associated with this instance's roles
-node.id=1
+node.id=2
```

Comment:
This isn't strictly necessary because the id space for brokers and controllers is distinct. I guess it's fine to do this though, to avoid confusion?